src/Repository/JobsRepository.php

```diff
@@ -20,47 +20,125 @@ public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, Jobs::class);
     }
-
-    public function save(Jobs $entity, bool $flush = false): void
+    
+    /**
+     * save
+     *
+     * @param  mixed $entity
+     * @return bool
+     */
+    public function save(Jobs $entity): bool
     {
         $this->getEntityManager()->persist($entity);
+        $this->getEntityManager()->flush();
 
-        if ($flush) {
-            $this->getEntityManager()->flush();
-        }
+        //if previous flush method is succesfull the $entity->getId() will return the id of the inserted company
+        //if previous flush method is not succesfull $entity->getId() will return NULL
+        return $entity->getId() > 0; // if null willreturn false
     }
-
-    public function remove(Jobs $entity, bool $flush = false): void
+    
+    /**
+     * remove
+     *
+     * @param  mixed $entity
+     * @return void
+     */
+    public function remove(Jobs $entity): void
     {
         $this->getEntityManager()->remove($entity);
+        $this->getEntityManager()->flush();
+    }
+       
+    /**
+     * update
+     *
+     * @param  mixed $id
+     * @param  mixed $params
+     * @return int
+     */
+    public function update(int $id, array $params): int
+    {
+        //create a new query builder object
+        $queryBuilder = $this->createQueryBuilder('j');
+
+        //prepare an update statement
+        $nbUpdatedRows = $queryBuilder->update()
```

Comment:
check if the name, description, active and priority are sent in the $params array. If one of them is missing we will get an array key missing error.
The query builder must be constructed like:
![image](https://user-images.githubusercontent.com/19302124/207569142-42cfa8ad-c71e-44e7-b394-036788172d05.png)